scripts/install.sh

```diff
@@ -569,4 +570,7 @@ function main() {
     report_install_success
 }
 
-main "$@"
+# tee logs into /var/log/kurl/install-<date>.log and stdout
+mkdir -p /var/log/kurl
+main "$@" 2>&1 | tee -a /var/log/kurl/install-$(date +"%m-%d-%Y-%H-%M-%S").log
```

Comment:
Wouldn't it be better to go with the format here? (`date +"%Y-%m-%dT%H-%M-%S"`) biggest to smallest...

Reply:
Done